api/src/main/java/org/openmrs/Person.java

```diff
@@ -325,6 +327,29 @@ public void setCauseOfDeath(Concept causeOfDeath) {
 		this.causeOfDeath = causeOfDeath;
 	}
 	
+	/**
+	 * This method returns the value of the causeOfDeathNonCoded attribute
+	 * @return cause of person's death non coded
```

Comment:
It would be better not to make the description be a mere method name with spaces. For instance, instead of "cause of person's death non coded", why not something like "non coded cause of death"?

Reply:
Likewise, instead of "This method returns the value of the causeOfDeathNonCoded attribute" why not something like "Gets the non coded cause of death"?

Reply:
It would also be great to be a line after the method description.